app/views/hyrax/autopopulation/dashboard/work_fetchers/index.html.erb

```diff
@@ -0,0 +1,79 @@
+
```

Comment:
Could this file be reformatted? There are quite a few extra break returns, commented out code and inconsistent indentations.